src/script/components/MessagesList/Message/ContentMessage/ContentMessage.tsx

```diff
@@ -62,14 +64,15 @@ export interface ContentMessageProps extends Omit<MessageActions, 'onClickResetS
   quotedMessage?: ContentMessage;
   selfId: QualifiedId;
   isMsgElementsFocusable: boolean;
+  messageRepository: MessageRepository;
```

Comment:
Humm the content message component should not know about the `messageRepository`. It's doesn't belong in this layer. 
The `MessageWrapper` on the other hand already knows about the `MessageRepository` and should be the one calling the method. 

Instead this component should be given a  callback `onClickReaction` and the parent (`MessageWrapper`) should make the call :)